docusaurus/docs/installation.md

```diff
@@ -114,8 +114,34 @@ Reassure CLI will automatically try to detect your source code branch name and c
 ```sh
 yarn reassure --branch [branch name] --commit-hash [commit hash] 
 ```
+## CI setup
+To make setting up the CI integration and all prerequisites more convenient, we have prepared a CLI command which will generate all necessary templates for you to get started with. 
 
-## Write performance testing script
+Simply run:
+```bash
+yarn reassure init --logLevel <verbose|default|silent>
+```
+This will generate the following file structure
+```
+├── <ROOT>
+│   ├── .reassure
+│   │   └──  reassure-tests.sh
+│   ├── dangerfile.ts (conditional)
+│   ├── dangerfile.reassure.ts (conditional)
+│   └── .gitignore (conditional)
+```
+
+#### `--logLevel` (optional)
```

Comment:
Seems like logLevel is related to all CLI commands, can we move it some common params section?

Reply:
Can we use something like `--verbose` and `--silent` rather than `--logLevel`? This seems to be more common approach for CLI tools.

Reply:
done